modules/planning/open_space/trajectory_smoother/distance_approach_ipopt_interface.h

```diff
@@ -30,6 +30,12 @@
 #include "adolc/adolc_sparse.h"
 #include "adolc/adouble.h"
 
+// TO-DO[runxin]: still have issue with adolc parallel
```

Comment:
@deidaraho Is this TODO still valid?

Reply:
Yes, still there. I need more checking time for this.